docs/Microservices/overview.md

```diff
@@ -55,9 +74,9 @@ With the established CIRA channel, Open AMT Cloud Toolkit enables the administra
 - reset
 - reset to BIOS
 
-For more information about power states and REST APIs, see Power States for more details. 
+For more information about power states and REST APIs, see [Intel® AMT Implementation and Reference Guide](https://software.intel.com/sites/manageability/AMT_Implementation_and_Reference_Guide/default.htm?turl=WordDocuments%2Fchangesystempowerstate.htm) for more details. 
```

Comment:
@kimberlyxdavis I would tweak this a bit:
"For more information about the power states supported by the REST APIs, see ..."

Reply:
Yes, that's fine.